cudatoolkit-dev/meta.yaml

```diff
@@ -1,5 +1,6 @@
 {% set name = "cudatoolkit-dev" %}
-{% set version = "9.2" %}
+{% set version = "9.2.148_396.37" %}
```

Comment:
@scopatz, @xmnlab, 

For linux, the cudatoolkit version is a result of the actual version (`9.2.148`) + the CUDA driver's version (`396.37`)
Due to this difference between linux and other OSes, I was wondering if you could have something like this in `meta.yaml`:
`{% set version = "9.2.148" %} [win, osx]`
`{% set version = "9.2.148_396.37" %} [linux]`

Reply:
For cudatoolkit I changed to just 9.2 .. and I am renaming the file using 'fn' for all platforms so in build.sh or bld.bat I just need 9.2 information

Reply:
@xmnlab, Thanks! I can see how this approach would work in your case since you are packaging everything you need during the build phase. I am going to look for ways to solve this issue in cudatoolkit-dev.

Reply:
Those selectors are totally fine